src/gossip/service.zig

```diff
@@ -1843,6 +1873,11 @@ pub const GossipService = struct {
         const gossip_table, var gossip_table_lock = self.gossip_table_rw.writeWithLock();
         defer gossip_table_lock.unlock();
 
+        // NOTE: this works rn because we only push contact_info whose 'deinit'
+        // method does not care about the given allocator, HOWEVER, this deinit allocator
+        // my not be correct for future values which are pushed into the queue.
+        const deinit_allocator = self.gossip_value_allocator;
```

Comment:
this sounds like a bug waiting to happen. should we make a ticket to address this with a proper solution?

Reply:
great point - https://github.com/orgs/Syndica/projects/2/views/10?pane=issue&itemId=90277798